src/lsstseries/ensemble.py

```diff
@@ -475,7 +580,14 @@ def from_parquet(
 
             # Generate an object table from the source table, then merge
             generated = self._generate_object_table()
-            self._object = file_table.merge(generated, how="right", on=[self._id_col])
+            self._object = file_table.merge(
+                generated, how="right", on=[self._id_col], suffixes=("_original", None)
+            )
+
+            # Drop overwritten columns
+            overwritten_cols = [col for col in list(self._object.columns) if "_original" in col]
+            self._object = self._object.drop(overwritten_cols, axis=1)
```

Comment:
I think we are repeating a lot of the logic from _sync_tables here. Would it be possible to replace this entire block with something like:

```
if object_file:  # read from parquet files
    self._object = self._generate_object_table()
self._sync_tables()
```

Reply:
This is a good idea, implemented.